web/concrete/controllers/single_page/dashboard/system/multilingual/setup.php

```diff
@@ -83,10 +83,6 @@ public function load_icon()
         $ch = Core::make('multilingual/interface/flag');
         $msCountry = $this->post('msCountry');
 
-        if (!$msCountry) {
```

Comment:
Returning here actually broke the page, if the region isn't found then it
prints "None" now instead of an exception that wasn't displayed.

On Tue, May 5, 2015, 2:57 PM Andrew Embler <notifications@github.com> wrote:

> In
> web/concrete/controllers/single_page/dashboard/system/multilingual/setup.php
> <https://github.com/concrete5/concrete5-5.7.0/pull/2378#discussion_r29700786>
> :
>
> > @@ -83,10 +83,6 @@ public function load_icon()
> >          $ch = Core::make('multilingual/interface/flag');
> >          $msCountry = $this->post('msCountry');
> >
> > -        if (!$msCountry) {
>
> Why was this removed?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/concrete5/concrete5-5.7.0/pull/2378/files#r29700786>.
>